File: PingPong.java
import java.io.Console;

public class PingPong {

  public static void main(String[] args) {
    Console myConsole = System.console();

    System.out.println("Pick any number!");
    String stringNumberPicked = myConsole.readLine();
    Integer integerNumberPicked = Integer.parseInt(stringNumberPicked);

    Integer currentNum = 0;
    System.out.println(currentNum);
    currentNum = currentNum + 1;

    while(currentNum <= integerNumberPicked) {
      if((currentNum % 5 == 0) && (currentNum % 3 == 0)) {
        System.out.println("ping-pong");
      }
      else if(currentNum % 5 == 0) {
        System.out.println("pong");
      }
      else if(currentNum % 3 == 0) {
        System.out.println("ping");
      }
      else {
        System.out.println(currentNum);
      }
      currentNum = currentNum + 1;
    }

    }
}

File: PingPong.js
function myFunction() {

var stringNumberPicked = prompt("Pick a whole number greater than zero!");
var integerNumberPicked = parseInt(stringNumberPicked);
var currentNum = 0;

if(integerNumberPicked <= 0) {
  document.write("Please enter a whole number greater than zero. Reload page to try again!");
}
else {
  document.write(currentNum + "<br>");
  currentNum = currentNum + 1;
  while(currentNum <= integerNumberPicked) {
    if((currentNum % 5 == 0) && (currentNum % 3 == 0)) {
      document.write("<b>PING-PONG</b>");
    }
    else if(currentNum % 5 == 0) {
      document.write("<b>pong</b>");
    }
    else if(currentNum % 3 == 0) {
      document.write("<b>ping</b>");
    }
    else {
      document.write(currentNum);
    }

  currentNum = currentNum + 1;
  document.write("<br>")
  }
  document.write("<br><br><b>Reload page to play again!</b>")
}
}
